app/views/declaration/transport_leaving_the_border.scala.html

```diff
@@ -57,8 +53,9 @@
                 RadioOption("Border_PostalOrMail", PostalConsignment.value, messages("declaration.transport.leavingTheBorder.transportMode.postalOrMail")),
                 RadioOption("Border_FixedTransportInstallations", FixedTransportInstallations.value, messages("declaration.transport.leavingTheBorder.transportMode.fixedTransportInstallations")),
                 RadioOption("Border_InlandWaterway", InlandWaterway.value, messages("declaration.transport.leavingTheBorder.transportMode.inlandWaterway")),
-                RadioOption("Border_Unknown", Unknown.value, messages("declaration.transport.leavingTheBorder.transportMode.unknown"))
-            )
+                RadioOption("Border_Unknown", Unknown.value, messages("declaration.transport.leavingTheBorder.transportMode.unknown")),
+                RadioOption("Border_Empty", Empty.value, messages("declaration.transport.leavingTheBorder.transportMode.empty"))
+            ).filterNot(radioOption => request.declarationType != DeclarationType.CLEARANCE && radioOption.value.contains(Empty.value))
```

Comment:
Why we need to filter it by dec type and this Empty.value?

Reply:
The "i dont know" answer is only applicable for clearance requests.   It's the new "optional" answer.

Reply:
Thank you Steve!